### PR TITLE
Adds Wehny

### PR DESCRIPTION
**Pull request by Kadeo64**
## About the PR

Adds Wehny, a horrible, horrible abomination of weh and rouny. It's an admin only spawn.
## Why / Balance
Admeme xeno with 4000 health (3900 till crit) and 50 blunt damage at 2 swing speed. It's also double the default runner's speed. good luck.
## Technical details
parented from runner.yml with values changed
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
god

### CODE REVIEW

**Comment by snebl** [comment]
He needs to be able to make a weh sound imo.

**Comment by USMP-lancer** [comment]
while your at it, add a glass rouny with 90% reflect chance but has literally 1 health

**Comment by a-person5660** [comment]
> while your at it, add a glass rouny with 90% reflect chance but has literally 1 health

it would die in 10 shots on average, you would want much more for it to survive anything.

File: Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/wehny.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    sprite: _RMC14/Mobs/Xenonids/Wehny/wehny.rsi
+  - type: MobThresholds
+    thresholds:
+      0: Alive
+      3900: Critical
+      4000: Dead
+  - type: XenoEvolution
+    max: 300
+    evolvesTo:
+    - CMXenoWehny
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/wehny.yml:17]:
```suggestion
  - type: XenoEvolution
    max: 300
    evolvesTo: [ ]
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

**Comment by QuietlyWhisper** [comment]
You should add rouny too :)

**Comment by Kadeo64** [comment]
> You should add rouny too :)

i'll do rouny after this probably

**Review by QuietlyWhisper — APPROVED** [review]
I'm sure it's fine

**Pull request merged**